### DISCUSSION TITLE
Welcome to relic_bazaar Discussions!

### DISCUSSION THREAD

**Discussion by himanshusharma89** [himanshusharma89/relic_bazaar#53] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ishandeveloper** [comment]
Heya hackers!

I'm Ishan, one of the collaborators on this project. Feel free to hit me up, or drop a message in case you need any help : )

#### Cheers & Happy Hacking ✨

**Comment by himanshusharma89** [comment]
Hello everyone!

So it is the 8th of March and the coding period starts for the GSSOC'21. We'll start assigning the issues and you can start coding 💙

**Comment by aayushbisen** [comment]
Hello team 👋, I voted for this project in Flutter Hackathon 🎯 . 
Great to see it here in GSSoC. 😄 
Looking forward to helping out. 💪

**Comment by aayushbisen** [comment]
![image](https://user-images.githubusercontent.com/41341387/110451082-dab59280-80e9-11eb-8cc9-40785a9307f7.png)

I was wondering whether the two search icons in the search bar might be confusing or not. What do you guys think?

I think only trailing icon makes sense, which will act as button in future.

**Reply by himanshusharma89** [reply]
Oh yeah, that makes sense. Go ahead create an issue and assign it to yourself. I think issue creators can assign themselves.

**Reply by aayushbisen** [reply]
Created the issue, but it seems that only project collaborators can assign themselves in issues.

**Reply by himanshusharma89** [reply]
Okay, I'll assign it to you.

**Comment by saurabhraj042** [comment]
Hey everyone, I came here via GSSoC .I have set-up the project locally and I'm going through the codebase for now.I'm very new to open source so any advice for me, also I'm looking for a good first issue.

**Reply by himanshusharma89** [reply]
Hey @saurabhraj042! Just a bit of advice, always create PR from a different branch than the master. Also, there are some UI issues, you can take them or you can create some issues yourself if you face any.

**Reply by saurabhraj042** [reply]
hi @himanshusharma89 I was going through the  HomePage and inside it the DrawerWidget listitems were lacking the onTap method.

**Reply by himanshusharma89** [reply]
Yeah, this will be done later when all the scenes are created

**Reply by aayushbisen** [reply]
Hello Saurabh, one good-first-issue I can think of is optimizing cart screen layout in landscape mode. You can see the issue yourself by going to cart screen and rotating the device.

Also @himanshusharma89 Do you plan to support landscape mode in the app?

**Reply by saurabhraj042** [reply]
@aayushbisen I just saw the issue and I think I can work on this, Can you please assign this to me ?

**Reply by himanshusharma89** [reply]
Yeah, @aayushbisen we can have that. I'll create those issues as soon as possible.

**Reply by saurabhraj042** [reply]
@himanshusharma89 Please assign it to me,

**Comment by Samdid** [comment]
hey @himanshusharma89, I am currently resolving the required login screen changes but I am  having some issues with my PC so I want little bit more time to fix that, is it ok to you?

**Reply by himanshusharma89** [reply]
Yeah, no issues.

**Reply by Samdid** [reply]
Ok thanks
